modules/ingestor/src/graph/sbom/spdx.rs

```diff
@@ -72,6 +72,25 @@ fn suppliers(sbom: &SPDX) -> Vec<String> {
     Vec::from_iter(result)
 }
 
+/// Extract versions for a SPDX SBOM by collecting versions of describing packages
+fn versions(sbom: &SPDX) -> Vec<String> {
+    // packages describing the SBOM
+    let describing = describing_packages(sbom);
+
+    // collect versions for matching packages
+    let mut result = HashSet::new();
+    for p in &sbom.package_information {
+        if !describing.contains(p.package_spdx_identifier.as_str()) {
+            continue;
+        }
+        if let Some(version) = &p.package_version {
```

Comment:
It's ok as it is. There's another way this should work, just that you know, and maybe you already do:

```rust
result.extend(p.package_version.clone());
```

`Option<T>` is an iterable (zero or one item), and `extend` can accept any iterable.